samples/audioProfile.js

```diff
@@ -14,43 +14,38 @@
  */
 
 'use strict';
+function synthesizeSpeechAsync(request = {}) {
```

Comment:
Make this an `async` function

Reply:
i removed this function, to maintain require statements and client creations with in a respective functions itself.